### PR TITLE
Add selected instrument urns to systemcard

### PR DESCRIPTION
**Pull request by ChristopherSpelt**
# Description

Adds selected instruments to system card. This is needed because the system card needs to contains URNs of instruments that need to be executed to fill it. This field is also assumed to be present in this PR: https://github.com/MinBZK/tad/pull/72

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x]  I have tested the changes locally and verified that they work as expected.
- [x]  I have followed the project's coding conventions and style guidelines.
- [x]  I have rebased my branch onto the latest commit of the main branch.
- [x]  I have squashed or reorganized my commits into logical units.
- [x]  I have read, understood and agree to the Developer Certificate of Origin below, which this project utilizes.

## Developer Certificate of Origin

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: docs/projects/tad/reporting-standard/latest.md [hunk 1]
```diff
@@ -73,10 +73,12 @@ A `system_card` contains the following information.
     4. `author` (OPTIONAL, string). Name of person that initiated the transformations.
 
 3. `name` (OPTIONAL, string). Name used to describe the system.
-4. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+4. `instrument_urns` (OPTIONAL, list[str]). List of Uniform Resource Names of instruments from
```
Comment (berrydenhartog) [inline docs/projects/tad/reporting-standard/latest.md:76]: Why instrument_urns and not just instruments?

Comment (ChristopherSpelt) [reply docs/projects/tad/reporting-standard/latest.md]: Processed your suggestion.

**Review by berrydenhartog — APPROVED** [review]
LGTM

File: docs/projects/tad/reporting-standard/latest.md [hunk 2]
```diff
@@ -73,10 +73,12 @@ A `system_card` contains the following information.
     4. `author` (OPTIONAL, string). Name of person that initiated the transformations.
 
 3. `name` (OPTIONAL, string). Name used to describe the system.
-4. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+4. `instruments` (OPTIONAL, list[str]). List of Uniform Resource Names of instruments from
```
Comment (robbertbos) [inline docs/projects/tad/reporting-standard/latest.md:76]: @ChristopherSpelt I would propose to make this a list of objects i.s.o. of a list of strings. If we make this change we could include additional information such as `instrument_version` and `required`.

File: mkdocs.yml [hunk 3]
```diff
@@ -85,15 +85,16 @@ nav:
...
         - projects/amt/reporting-standard/0.1a5.md
         - projects/amt/reporting-standard/0.1a4.md
         - projects/amt/reporting-standard/0.1a3.md
         - projects/amt/reporting-standard/0.1a2.md
         - projects/amt/reporting-standard/0.1a1.md
         - projects/amt/comparison.md
-      - ADRs for AMT:
-        - ... | flat | projects/amt/adrs/*
+      - ADRs for TAD:
+        - ... | flat | projects/tad/adrs/*
```
Comment (robbertbos) [inline mkdocs.yml:97]: I guess we need a fix here. @ChristopherSpelt can you check this?

File: docs/projects/amt/reporting-standard/latest.md [hunk 4]
```diff
@@ -73,10 +73,16 @@ A `system_card` contains the following information.
...
 
 3. `name` (OPTIONAL, string). Name used to describe the system.
-4. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+4. `instruments` (OPTIONAL, list). List of instruments from
+the [instrument register](https://github.com/MinBZK/instrument-registry) that must be executed to fill this system card.
+There can be multiple instruments. For each instrument the following fields are present.
+    1. `urn` (OPTIONAL, string). Uniform Resource Names of the instrument.
+    2. `version` (OPTIONAL, string). The version of the instrument.
+    3. `required` (OPTIONAL, boolean). Specifies whether the instrument is required to be executed or not.
+5. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
```
Comment (robbertbos) [inline docs/projects/amt/reporting-standard/latest.md:82]: Markdown "code" quality suggestion:

```suggestion
4. `instruments` (OPTIONAL, list). List of instruments from
   the [instrument register](https://github.com/MinBZK/instrument-registry) that must be executed to fill this system
   card. There can be multiple instruments. For each instrument the following fields are present.

    1. `urn` (OPTIONAL, string). Uniform Resource Names of the instrument.
    2. `version` (OPTIONAL, string). The version of the instrument.
    3. `required` (OPTIONAL, boolean). Specifies whether the instrument is required to be executed or not.

5. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
```

File: docs/projects/amt/reporting-standard/latest.md [hunk 5]
```diff
@@ -73,10 +73,16 @@ A `system_card` contains the following information.
     4. `author` (OPTIONAL, string). Name of person that initiated the transformations.
 
 3. `name` (OPTIONAL, string). Name used to describe the system.
-4. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+4. `instruments` (OPTIONAL, list). List of instruments from
+the [instrument register](https://github.com/MinBZK/instrument-registry) that must be executed to fill this system card.
+There can be multiple instruments. For each instrument the following fields are present.
+    1. `urn` (OPTIONAL, string). Uniform Resource Names of the instrument.
```
Comment (robbertbos) [inline docs/projects/amt/reporting-standard/latest.md:79]: @ChristopherSpelt, when an instrument object is added, shall we make the `urn` required?

**Review by robbertbos — CHANGES_REQUESTED** [review]

File: docs/projects/amt/reporting-standard/0.1a7.md [hunk 6]
```diff
@@ -0,0 +1,508 @@
+# :label: 0.1a7
+
+This document describes the Transparency of Algorithmic Decision making (TAD) Reporting Standard.
```
Comment (laurensWe) [inline docs/projects/amt/reporting-standard/0.1a7.md:3]:
```suggestion
This document describes the Algorithm Management Toolkit (AMT) Reporting Standard.
```

File: docs/projects/amt/reporting-standard/0.1a7.md [hunk 7]
```diff
@@ -0,0 +1,508 @@
...
+
+For reproducibility, governance, auditing and sharing of algorithmic systems it is essential to have a reporting
+standard so that information about an algorithmic system can be shared. This reporting standard describes how
+information about the different phases of an algorithm's life cycle can be reported. It contains, among other things,
+descriptive information combined with information about the technical tests and assessments applied.
+
+!!! warning "Disclaimer"
+
+    The TAD Reporting Standard is work in progress. This means that the current standard is probably suboptimal and will
```
Comment (laurensWe) [inline docs/projects/amt/reporting-standard/0.1a7.md:12]:
```suggestion
    The AMT Reporting Standard is work in progress. This means that the current standard is probably suboptimal and will
```

**Review by laurensWe — COMMENTED** [review]
Just the renaming of TAD to AMT, for the rest I completely agree

**Pull request merged**